apps/web/src/design-system/icons/general/ChatFilled.tsx

```diff
@@ -13,7 +13,7 @@ export function ChatFilled(props: React.ComponentPropsWithoutRef<'svg'>) {
       {...props}
     >
       <path
-        fill="url(#paint0_linear_528_4335)"
+        fill={theme.colorScheme === 'light' ? colors.B60 : 'white'}
```

Comment:
@Aniketsahu28 you do not have the theme or the colors import here. Am I missing something?

Reply:
I will make the changes and upload screenshot in a while

Reply:
Will you please check  is it fine ?

Reply:
@Aniketsahu28 are you able to run this repo locally ?

Reply:
> @Aniketsahu28 are you able to run this repo locally ?

I tried 3+ times, but I am not able to run it locally 🥲 @Aniketsahu28 @rjangid9080 @ainouzgali 

![Screenshot 2023-10-01 153937](https://github.com/novuhq/novu/assets/96189881/41a7d8b1-7330-47df-903b-aff193a90ad9)